x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_status_filter.tsx

```diff
@@ -59,22 +49,29 @@ export const RuleStatusFilter = (props: RuleStatusFilterProps) => {
     setIsPopoverOpen((prevIsOpen) => !prevIsOpen);
   }, [setIsPopoverOpen]);
 
-  const renderClearAll = () => {
-    return (
-      <div>
-        <EuiButtonEmpty
-          style={{
-            width: '100%',
-          }}
-          size="xs"
-          iconType="crossInACircleFilled"
-          color="danger"
-          onClick={() => onChange([])}
-        >
-          Clear all
-        </EuiButtonEmpty>
-      </div>
-    );
+  const renderRuleStateOptions = (status: 'enabled' | 'disabled' | 'snoozed') => {
+    if (status === 'enabled') {
+      return (
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.ruleDetails.ruleStateFilter.enabledOptionText"
+          defaultMessage="Rule is enabled"
+        />
+      );
+    } else if (status === 'disabled') {
+      return (
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.ruleDetails.ruleStateFilter.disabledOptionText"
+          defaultMessage="Rule is disabled"
```

Comment:
Ditto re shorter is better:
```suggestion
          defaultMessage="Disabled"
```